java/code/src/com/redhat/rhn/domain/contentmgmt/modulemd/ModuleStreams.java

```diff
@@ -45,4 +46,13 @@ public String getDefaultStream() {
     public List<String> getStreams() {
         return streams;
     }
+
+    @Override
+    public String toString() {
+        streams.sort(Comparator.naturalOrder());
+        if (defaultStream == null) {
+            return String.join(", ", streams);
+        }
+        return String.join(", ", streams).replaceAll(defaultStream, "<b>" + defaultStream + "</b>");
+    }
```

Comment:
With this PR, we're introducing a feature that is specific to the web UI, but this class and `toString` in general has a much wider context, so it'd be better to move this formatting elsewhere. `toString` is expected to be used in many different contexts like APIs, CLI output, logs, stack traces, etc.

I think it'd be much better to do this formatting in the controller + JSP.

For example, if the sorting is required, it should be done in `ApplicationStreamsAction` to prepare it for JSP, and the `<b>` tags should be added inside the JSP file directly. Otherwise, for example, the `<b>` tags would also be searched with filtering. This can be done using some conditionals in JSP using the [core tag set](https://docs.oracle.com/javaee/5/jstl/1.1/docs/tlddocs/c/tld-summary.html).

My suggestion would be to add a `<c:forEach>` loop in the JSP and loop through the `streams` list and print each, and conditionally print with the `<b>` tags if the current item is equal to `defaultStream`. This approach separates the concern of formatting from the rest of the code.

For an even nicer UI, you can print each stream as a [Bootstrap label](https://getbootstrap.com/2.3.2/components.html#labels-badges) (blue for default, gray for others).